sap/xssec/security_context.py

```diff
@@ -270,6 +270,7 @@ def _set_ext_attr(self, jwt_payload):
             self._properties['subdomain'] = ext_attr.get('zdn')
             self._logger.debug('Obtained subdomain: %s.', self._properties['subdomain'])
 
+            # set subaccount_id to the zone_id as a workaround, if subaccount id is not available.
             if 'subaccountid' in ext_attr:
                 self._properties['subaccount_id'] = ext_attr.get('subaccountid')
```

Comment:
I don't know how python works here.
What happens if the json looks like this:
```
{
  ext_attr: {
      subaccountid: null
  }
}
```
In node.js I make sure that in this case also self._properties['subaccount_id'] = jwt_payload['zid']

But if this could really happen, I don't know. :-)

Reply:
ok, I'm also checking if it's null now